.ci/install_sam_cli.sh

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
This is empty? I think accidental, since I see pip install it.

Reply:
ah, yes. It's not needed anymore